elasticsearch.md

```diff
@@ -542,6 +542,20 @@ If you happen to get a `java.io.IOException: failed to decrypt safe contents ent
 (Credits for the original JKS tutorial to [Maximilian Boehm](https://maximilian-boehm.com))
 
 
+### Request handling during ES startup
+
+Each incoming request to the Elasticsearch node passes to the installed plugin. During Elasticsearch node startup, the plugin rejects incoming requests until it starts. The plugin rejects such requests with the `403` forbidden responses by default.
+To override this behavior, append to  **elasticsearch.yml**
+
+```
+readonlyrest.not_started_response_code: 403
```

Comment:
we are writing about changing the default codes but we show an example with the default ones.